builder/builder.go

```diff
@@ -117,8 +124,15 @@ func (b *Builder) CreateDefaultConfig(localrpms bool) error {
 		return err
 	}
 
+	// Patch all default path prefixes to PWD
 	data := strings.Replace(string(raw), "/home/clr/mix", pwd, -1)
+
+	// Patch default directory for version files
+	// TODO: Clean this up once it's not needed
+	data = strings.Replace(data, "VERSIONS_PATH="+pwd+"\n", "VERSIONS_PATH="+filepath.Join(pwd, b.Mixerdir)+"\n", -1)
+
 	if localrpms {
```

Comment:
Is it worth thinking about changing this to use the template system? Probably not for this PR, but future work.